packages/server/lib/crons/exportMetrics.ts

```diff
@@ -0,0 +1,46 @@
+import * as cron from 'node-cron';
```

Comment:
Not sure it belongs to server (or jobs tbh) but please mind all the pods will execute this cron job so it's going to be executed more than once

Reply:
from a metrics point of view it doesn't matter since the value we are gonna be interested in is the last one.
Do you think it is worth locking anyway?

Reply:
not sure I just wanted to point out, worst case all of them don't report the same value but you can use `max` I guess